Truck.js

```diff
@@ -1,6 +1,34 @@
 var Car = require('./Car.js');
 
 //Implement your Truck here
-function Truck() {};
+function Truck(make, model, year, color, capacity) {
+	Car.call(this, make, model, year, color, 3);
+	this.capacity = capacity || 0;
+	this.cargo = 0;
+};
+
+Truck.prototype = Object.create(Car.prototype);
+Truck.prototype.constructor = Truck;
+
+Truck.prototype.load = function(pounds) {
+	var takeLoad = false;
+	if (pounds + this.cargo <= this.capacity) {
+		this.cargo += pounds;
+		takeLoad = true;
+		console.log("loaded " + pounds + "lbs of cargo.");
+	}
+	return takeLoad;
+}
+
+Truck.prototype.unload = function(pounds) {
+	var removeLoad = false;
```

Comment:
I think you're adding a bit extra work for yourself here, you probably can just return true or false without storing those booleans in a variable.